config/phenomenal_tools2container.yaml

```diff
@@ -282,6 +282,13 @@ assignment:
   docker_tag_override: v1.0_cv0.2.30
   max_pod_retrials: 3
   tools_id:
+ - bruker2batman
```

Comment:
You're mixing the isodyn entry with the bruker2batman entry, please be careful. I'll fix this myself.